tests/test_fio_calc.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from click.testing import CliRunner
+
+from fiona.fio.calc import calc
+from .fixtures import feature_seq
+
+
+def test_fail():
+    runner = CliRunner()
+    result = runner.invoke(calc,
+                           ["TEST", "f.properties.test > 5"],
+                           "{'type': 'no_properties'}")
+    assert result.exit_code == 1
```

Comment:
@perrygeo While this expression does fail because the input feature lacks a `properties` key, it is currently failing because the input is invalid JSON.  The quote type on the input JSON needs to be switched to: `'{"type": "no_properties"}'`.

Adding an `assert False, result.output` right below this line shows the failure:

```console
E       AssertionError: Exception caught during processing
E         Traceback (most recent call last):
E           File "/Users/wursterk/code/perrygeo-fiona/fiona/fio/calc.py", line 62, in calc
E             for i, obj in enumerate(source):
E           File "/Users/wursterk/code/perrygeo-fiona/fiona/fio/helpers.py", line 30, in gen
E             yield json.loads(first_line)
E           File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/__init__.py", line 319, in loads
E             return _default_decoder.decode(s)
E           File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/decoder.py", line 339, in decode
E             obj, end = self.raw_decode(s, idx=_w(s, 0).end())
E           File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/decoder.py", line 355, in raw_decode
E             obj, end = self.scan_once(s, idx)
E         json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 2 (char 1)
E         Aborted!
```